src/test/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTaskTest.java

```diff
@@ -486,6 +494,44 @@ public void testCheckerException() throws IOException {
         }
     }
 
+    @Test
+    public final void testExecuteLogOutput() throws Exception {
+        final CheckstyleAntTaskLogStub antTask = new CheckstyleAntTaskLogStub();
+        final URL url = new File(getPath(CONFIG_FILE)).toURI().toURL();
+        antTask.setProject(new Project());
+        antTask.setConfigURL(url);
+        antTask.setFile(new File(getPath(FLAWLESS_INPUT)));
+
+        mockStatic(System.class);
+        when(System.currentTimeMillis()).thenReturn(1L);
+
+        antTask.execute();
+
+        final List<MessageLevelPair> expectedList = Arrays.asList(
+                new MessageLevelPair("checkstyle version ", 3),
```

Comment:
I think numbers should be replaced with `Project.XXXX` to make it more human friendly.